main_test.go

```diff
@@ -41,13 +41,14 @@ func TestReadFromYaml_DuplicateReleaseName(t *testing.T) {
 	}
 }
 
-func makeRenderer(readFile func(string) ([]byte, error), env string) *twoPassRenderer {
+func makeRenderer(readFile func(string) ([]byte, error), env string, namespace string) *twoPassRenderer {
```

Comment:
this is a test helper, and I see that you do not use the parameter, what do you think about hard-coding it in the struct creation?

```
return &twoPassRenderer{
		reader:    readFile,
		env:       env,
		namespace: "namespace",
		filename:  "",
		logger:    logger,
		abs:       filepath.Abs,
	}